lewis/scripts/control.py

```diff
@@ -41,16 +41,20 @@ def show_api(remote, object_name):
     methods = []
 
     for member_name in dir(obj):
-        if not member_name[0] in ('_', ':'):
-            member = getattr(obj, member_name)
-
-            if callable(member):
-                methods.append(member_name)
+        if member_name[0] not in ('_', ':') and member_name not in dir(type(obj)):
+            methods.append(member_name)
 
     print('Type: {}'.format(type(obj).__name__))
-    print('Properties:')
+
+    print('Properties (current values):')
     for prop in sorted(properties):
-        print('\t{}'.format(prop))
+        try:
+            current_value = getattr(obj, prop)
+        except Exception as e:
+            current_value = 'Not accessible: {}'.format(e.message)
+
+        print('\t{}\t{}'.format(prop, current_value))
```

Comment:
Consider for this segment:
```python
    print('Properties (current values):')
    maxlen = len(max(properties, key=len))
    for prop in sorted(properties):
        try:
            current_value = getattr(obj, prop)
        except Exception as e:
            current_value = 'Not accessible: {}'.format(e.message)

        print('    {}    ({})'.format(prop.ljust(maxlen), current_value))

    print('Methods:')
    for method in sorted(methods):
        print('    {}'.format(method))
```
Changes are:
- `ljust` to align columns
- parentheses around value to mirror "Properties (current values):" header
- four spaces instead of tabs for less width and more consistency (length of tab depends on where you tab from)